lrauv_ignition_plugins/proto/dev_beam_state.proto

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Knit: File name is `dev_beam_state.proto` would probably be better to call it `dvl_beam_state.proto`